deployment/kubernetes/scripts/deploy-kind.sh

```diff
@@ -694,49 +692,60 @@ run_health_checks() {
         -H "Authorization: Bearer $auth_token" \
         "http://localhost:$http_port/api/ingress/v1/version" 2>/dev/null || echo "000")
 
+    # Test ROS-OCP Ingress API endpoint (this will fail if services aren't deployed, which is expected)
+    local ingress_response=$(curl -s -o /dev/null -w "%{http_code}" --connect-timeout 5 --max-time 15 "http://localhost:$http_port/api/ingress/v1/version" 2>/dev/null || echo "000")
+    
     if [ "$ingress_response" = "200" ] || [ "$ingress_response" = "401" ]; then
-        echo_success "ROS-OCP Ingress API is accessible via ingress on port $http_port (HTTP ${ingress_response})"
-    else
-        echo_error "ROS-OCP Ingress API is not accessible via ingress on port $http_port (HTTP ${ingress_response})"
-        failed_checks=$((failed_checks + 1))
-    fi
-
-    # Check if ROS-OCP API is accessible via ingress with authentication
-    local api_response=$(curl -s -o /dev/null -w "%{http_code}" \
-        -H "Authorization: Bearer $auth_token" \
-        "http://localhost:$http_port/status" 2>/dev/null || echo "000")
-
-    if [ "$api_response" = "200" ] || [ "$api_response" = "401" ]; then
-        echo_success "ROS-OCP API is accessible via ingress on port $http_port (HTTP ${api_response})"
-    else
-        echo_error "ROS-OCP API is not accessible via ingress on port $http_port (HTTP ${api_response})"
-        failed_checks=$((failed_checks + 1))
-    fi
-
-    # Check if Kruize is accessible via ingress with authentication
-    local kruize_response=$(curl -s -o /dev/null -w "%{http_code}" \
-        -H "Authorization: Bearer $auth_token" \
-        "http://localhost:$http_port/api/kruize/listPerformanceProfiles" 2>/dev/null || echo "000")
-
-    if [ "$kruize_response" = "200" ] || [ "$kruize_response" = "401" ]; then
-        echo_success "Kruize API is accessible via ingress on port $http_port (HTTP ${kruize_response})"
+        echo_success "✓ ROS-OCP Ingress API is accessible via ingress on port $http_port (HTTP ${ingress_response})"
     else
-        echo_error "Kruize API is not accessible via ingress on port $http_port (HTTP ${kruize_response})"
-        failed_checks=$((failed_checks + 1))
+        echo_info "ℹ ROS-OCP Ingress API not accessible (HTTP ${ingress_response}) - expected if services not deployed"
+    fi
+
+    # Check if ROS-OCP services are deployed
+    local ros_ocp_deployed=false
+    if kubectl get namespace ros-ocp >/dev/null 2>&1; then
```

Comment:
I think we can safely remove this checks, there should not be any ros-ocp namespace at this stage. WDYT?